Assets/Global/Scripts/RoomLogic/DoorManager.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using UnityEngine.SceneManagement;
+
+public class DoorManager : Reference
+{
+    public List<int> connectedRooms = new List<int>(); 
+    public List<GameObject> doors;
```

Comment:
can this be private? or if you use this in other scripts use public with [HideInInspector] (but i don't see any references)